R/three-legged-oauth.R

```diff
@@ -22,6 +36,9 @@ credentials_user_oauth2 <- function(scopes,
                                     app = gargle_app(),
                                     ...) {
   "!DEBUG trying credentials_user_oauth2"
+  if (missing(scopes)) {
+    scopes <- "email"
```

Comment:
Just put in args?

Reply:
I feel like I had a reason for doing this (it goes back to the ancient origins of this PR) but damn if I can remember. I'll make a comment so I re-examine when I'm really looking at general token fetching vs. just focused on oauth.